### PR TITLE
Airdrop claim

### PR DESCRIPTION
**Pull request by Tristyn**

### CODE REVIEW

File: src/common/associatedToken.ts [hunk 1]
```diff
@@ -111,7 +111,7 @@ export class AssociatedToken {
    * @returns {(TokenAccountInfo | undefined)}
    * @memberof AssociatedToken
    */
-  static useAux(connection: Connection | undefined, tokenAddress: PublicKey | undefined): AssociatedToken | undefined {
+  static useATA(connection: Connection | undefined, tokenAddress: PublicKey | undefined): AssociatedToken | undefined {
```
Comment (Tristyn) [inline src/common/associatedToken.ts:114]: These 'aux' functions are actually meant to load a non associated token account

File: src/rewards/airdrop.ts [hunk 2]
```diff
@@ -144,6 +162,23 @@ export interface AirdropTarget {
...
+
+  /// A description for this airdrop
+  short_desc: string
+
+  /// The bump seed needed to generate the airdrop account address
+  vault_bump: number
+
+  /// Airdrop settings
+  flags: BN
+}
```
Comment (Tristyn) [inline src/rewards/airdrop.ts:180]: Nice Nice Nice

You can use jsdoc
```suggestion
export interface AirdropCreateParams {
  /** The expiration time for the airdrop /
  expire_at: BN

  /** The stake pool that claimed rewards are deposited into. */
  stake_pool: PublicKey

  /** A description for this airdrop */
  short_desc: string

  /** The bump seed needed to generate the airdrop account address */
  vault_bump: number

  /** Airdrop settings */
  flags: BN
}
```

File: src/rewards/airdrop.ts [hunk 3]
```diff
@@ -245,4 +280,233 @@ export class Airdrop {
   getRecipient(wallet: PublicKey) {
     return this.targetInfo.recipients.find(recipient => wallet.equals(recipient.recipient))
   }
+
+  // claim
+  // TODO - now: airdrop_claim.rs
+
+  static async withClaim(
```
Comment (Tristyn) [inline src/rewards/airdrop.ts:287]: Nice work!

File: src/common/associatedToken.ts (hunk 1 above)
Comment (qiqicodes) [reply src/common/associatedToken.ts:118]: revert to useAux

File: src/rewards/airdrop.ts (hunk 2 above)
Comment (qiqicodes) [reply src/rewards/airdrop.ts:181]: AirdropCreatParams removed because it is not needed for current jet-gov integration

**Review by callensm — APPROVED** [review]

**Pull request merged**